parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -91,9 +91,28 @@ assignFilePermissions() {
     chmod 644 /etc/default/grub || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
     chmod 644 /etc/pam.d/su || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
     chmod 755 /etc/profile.d/CIS.sh || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
+    for filepath in /etc/crontab /etc/cron.hourly /etc/cron.daily /etc/cron.weekly /etc/cron.monthly /etc/cron.d; do
```

Comment:
There is no `/etc/at.allow` or `/etc/cron.allow` I assume?

Reply:
I think we should defer locking down cron to certain users until we figure out how to do that sustainably. I.e., we don't want to restrict to root, as arguably we *don't* want to encourage folks to use root. Basically, we need a definition of "authorized users".